src/components/theharvester/theharvester.tsx

```diff
@@ -132,13 +185,27 @@ const TheHarvester = () => {
         } catch (e: any) {
             setOutput(e.message);
         }
+
+        // Deactivate loading state
         setLoading(false);
     };
+
+    // Clears the output state utilising the callback function to revert the output to an empty string
     const clearOutput = useCallback(() => {
         setOutput("");
     }, [setOutput]);
+
     return (
+        <RenderComponent
+            title={title}
+            description={description}
+            steps={steps}
+            tutorial={tutorial}
+            sourceLink={sourceLink}
+        >
         <form onSubmit={form.onSubmit(onSubmit)}>
```

Comment:
You need to add the loadingModal and InstallationModal here. `
```suggestion
 {!loadingModal && (
                <InstallationModal
                    isOpen={opened}
                    setOpened={setOpened}
                    feature_description={description}
                    dependencies={dependencies}
                ></InstallationModal>
            )}
        <form onSubmit={form.onSubmit(onSubmit)}>
```